back/src/route/reviewController.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcryptjs');
+const { v4: uuidv4 } = require('uuid');;
+const {info, debug, warning, error} = require("../winston");
+const Review = require("../objects/review");
+const fileLabel = "reviewController"
+const reviewSQL = require("../database/ReviewSQL");
+
+
+module.exports = class ReviewController{
+    constructor(){
+        this.register.bind(this);
+    }
+    
+    async register(req, res){
+        const {spotId, comment, score, userId} = req.body;
+        //encrypt password
+        let salt = bcrypt.genSaltSync(10);
+        const review = new Review(uuidv4(), spotId, comment, score, userId);
```

Comment:
We need to make sure that the data exists before creating the Review. Right now the system allows us to create review with no spotId or comment.

<img width="452" alt="Screen Shot 2020-11-17 at 2 12 47 PM" src="https://user-images.githubusercontent.com/11179982/99349365-2e079900-28df-11eb-922e-4f0d6fee12bc.png">

Reply:
Request used: 
{
    "spotId" : "",
    "comment" : "", 
    "score" : 100, 
    "userId" : "TestingEmptyFields"
}